.circleci/behat-test.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+# Bail if required environment varaibles are missing
+if [ -z "$TERMINUS_SITE" ] || [ -z "$TERMINUS_ENV" ]
+then
+  echo 'No test site specified. Set TERMINUS_SITE and TERMINUS_ENV.'
+  exit 1
+fi
+
+if [ -z "$ADMIN_USERNAME" ] || [ -z "$ADMIN_PASSWORD" ]
+then
+	echo "No WordPress credentials specified. Set ADMIN_USERNAME and ADMIN_PASSWORD."
+	exit 1
+fi
+
+echo "::::::::::::::::::::::::::::::::::::::::::::::::"
+echo "Behat test site: $TERMINUS_SITE.$TERMINUS_ENV"
+echo "::::::::::::::::::::::::::::::::::::::::::::::::"
+echo
+
+# Exit immediately on errors
+set -ex
+
+# Create a backup before running Behat tests
+terminus -n backup:create $TERMINUS_SITE.$TERMINUS_ENV
+
+# Clear site cache
+terminus -n env:clear-cache $TERMINUS_SITE.$TERMINUS_ENV
+
+# Stash current WordPress username
+export WORDPRESS_USER_NAME=$ADMIN_USERNAME
+
+# Use a generic Pantheon user for testing
+export ADMIN_USERNAME='pantheon-ci-testing'
+
+# Setup the WordPress admin user
+terminus -n wp $TERMINUS_SITE.$TERMINUS_ENV -- user delete $ADMIN_USERNAME --yes
```

Comment:
Using the default username caused it to be deleted and/or password changed after testing. Switched to using a different username for Behat.